client/src/components/auth/LayoutMainPage.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <VContainer>
+        <VRow class="rowHeight-70">
```

Comment:
okay, we a step further. First of all, classes must be called in lower case. It is worth using BEM notation. The structure is next:

block__element--modifier

block - is top level class, if you use multiple words, than use "-" as divider
element - block can contain elements as children, but cannot contain elements of another blocks. But block can contain another blocks
modifier - any block or element can have a modifier, for example if you want to define some special color, so it would be block--color, or block__element--color

Pay attention, you cannot define modifier without block. For instance, in your case, you should ask yourself what logic reflects your row? So it seems it is header or logo-container, okay then:

class="logo-container"

then
.logo-container {
   height: 70px;
}

and do similar things for others classes